dGame/dComponents/SkillComponent.cpp

```diff
@@ -192,7 +196,15 @@ void SkillComponent::Interrupt() {
 	auto* combat = m_Parent->GetComponent<BaseCombatAIComponent>();
 	if (combat != nullptr && combat->GetStunImmune()) return;
 
-	for (const auto& behavior : this->m_managedBehaviors) behavior.second->Interrupt();
+	for (const auto& behavior : this->m_managedBehaviors) {
```

Comment:
Should the managed behaviors be cleared after this?

Reply:
The managed behaviors are need to be maintained and not cleared as clearing them would cause the desyncs players currently experience.  To match the client, end nodes fire off as to keep start nodes in sync with what the client would expect them to be in, however we dont want to interrupt anything that could be synced from a player because if we cleared those during the lag frames, players would experience the issues they currently see with skills not having a valid uid entry in the managed behaviors.